integration/spark2/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala

```diff
@@ -47,7 +47,7 @@ trait ServerInterface {
   /**
    * Get the cache size for the specified table.
    */
-  def showCache(tableName: String) : Array[String]
+  def showCache(tableNames: String) : Array[String]
```

Comment:
Please optimize the description.  table=> tables

Reply:
done